proxy/frontend_http_handler.go

```diff
@@ -91,6 +97,20 @@ func (h *FrontendHTTPHandler) serveHTTP(rw http.ResponseWriter, req *http.Reques
 	return err
 }
 
+func (h *FrontendHTTPHandler) copyAuthHeaders(req *http.Request) {
+	c, err := req.Cookie("token")
+	if err != nil || c == nil {
+		return
+	}
+
+	authHeader := req.Header.Get("Authorization")
+	if authHeader != "" {
+		return
+	}
+
+	req.Header.Set("Authorization", "Bearer "+base64.StdEncoding.EncodeToString([]byte("Bearer "+c.Value)))
```

Comment:
Is this supposed to have Bearer twice? Once in the encoded bit and once outside the encoded bit